patches/picolibc.patch

```diff
@@ -11,6 +11,13 @@ index c49856ee0..3e927ea75 100644
 +It's necessary to set this globally because inline functions in
 +libc++ headers call the GNU functions.'''
 +)
++
++conf_data.set('_PICOLIBC_CTYPE_SMALL', '0',
++        description: '''Disable small versions of ctype functions.
++Use this to use the classic, table based ctype implementation and not
```

Comment:
This text seems to be a description of what `_PICOLIBC_CTYPE_SMALL` is for. Wouldn't it be more useful to describe _why_ we want to set it in a non-default way? The purpose of the flag can be found out from picolibc itself.

Something like: "libc++ expects newlib-style ctype tables, and also expects support for locales and extended character sets, so picolibc's small ctype is not compatible with it".